DiscordChatExporter.Domain/Exporting/MediaDownloader.cs

```diff
@@ -16,13 +16,16 @@ internal partial class MediaDownloader
     {
         private readonly HttpClient _httpClient = Singleton.HttpClient;
         private readonly string _workingDirPath;
+
+        private readonly bool _reuseMedia;
         private readonly AsyncRetryPolicy _httpRequestPolicy;
 
         private readonly Dictionary<string, string> _pathMap = new Dictionary<string, string>();
 
-        public MediaDownloader(string workingDirPath)
+        public MediaDownloader(string workingDirPath, bool reuseMedia = false)
```

Comment:
Let's not use default parameters in internal stuff, so that we don't accidentally forget to set it.

Reply:
Good idea.